app/styles/modules/_home.scss

```diff
@@ -0,0 +1,202 @@
+#landing-page {
+  opacity: 0;
+}
+
+/* The following is a lightly modified version of the styles
+*  used by the fullpage plugin.
+*/
+
+#superContainer {
+  height: 100%;
+  position: relative;
+
+/* Touch detection for Windows 8 */
+-ms-touch-action: none;
```

Comment:
We should see if we need these vendor prefixes in this file or if auto-prefixer can auto-manage them for us.